common/stream-model.cpp

```diff
@@ -779,177 +773,242 @@ namespace rs2
             }
         }
 
+        if (show_metadata)
+            stream_model::draw_stream_metadata(timestamp, timestamp_domain, frame_number, profile, original_size, stream_rect);
 
+        ImGui::PopStyleColor(5);
```

Comment:
From when I understand this pops up colors that were pushed inside the function above, if that's true I would move it inside.
It looks weird to see pop when you don't have a push above

Reply:
I count all `PopStyleColor` and `PopStyleColor` functions and found that in the 779 line, we are popping styles from the
lines 673-678:
`ImGui::PushStyleColor(ImGuiCol_Text, light_grey);`
`ImGui::PushStyleColor(ImGuiCol_TextSelectedBg, white);`
`ImGui::PushStyleColor(ImGuiCol_Button, header_window_bg);`
`ImGui::PushStyleColor(ImGuiCol_ButtonHovered, header_window_bg);`
`ImGui::PushStyleColor(ImGuiCol_ButtonActive, header_window_bg);`

More of all if I write another value, not 5, I have a run time error on the style stack:

- With `ImGui::PopStyleColor(4);`

![image](https://user-images.githubusercontent.com/26552140/235419785-78868be5-0c23-40bd-a60e-f05a83fab639.png)

- With `ImGui::PopStyleColor(6);`

![image](https://user-images.githubusercontent.com/26552140/235419999-f3ab0449-9168-4875-93e7-608b13bac787.png)

Reply:
I understand that, 5 is needed.
But not where you have it.
I say let's move this pop(5) to inside the function that does the push.

Reply:
You are right it looks good